CoreDataAccessView/src/au/gov/asd/tac/constellation/views/dataaccess/plugins/utility/SelectAllPlugin.java

```diff
@@ -38,6 +41,8 @@
 @Messages("SelectAllPlugin=Select All")
 public class SelectAllPlugin extends SimpleQueryPlugin implements DataAccessPlugin {
 
+    private static final String ATTRIBUTE_ERROR = "The SelectAllPlugin could not successfully complete because it does not contain the attribute %s";
```

Comment:
I think it would be ideal to use more user-friendly language in the error message. Perhaps instead of "The SelectAllPlugin" you could simply say "Select All" and then when referring to the selected attributes say something like "the selected node attribute" rather than "the attribute VisualConcept.VertexAttribute.SELECTED"

Reply:
I changed it to Select All count not successfully complete because it does not contain the Select Vertex Attribute 'Selected'.
That way it seems understandable by the user but also helps direct them to the actual attribute name responsible.